README.md

```diff
@@ -1 +1,192 @@
-# to_do_list
\ No newline at end of file
+<a name="readme-top"></a>
+
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 To-Do list <a name="about-project"></a>
+
+- This website shows a sneak peek into the projects I have developed, showcasing my skills in HTML, CSS, javascript and webpack. With a passion for web development, I have honed my expertise in these technologies to create stunning and functional websites.
```

Comment:
- [ ] Please describe the project in this section to the readers per the requirement.